### PR TITLE
fix(i18n): Changed grammar

### PR DESCRIPTION
**Pull request by rakekniven**
Reported at Transifex.

### CODE REVIEW

File: apps/files/js/mainfileinfodetailview.js [hunk 1]
```diff
@@ -163,7 +163,7 @@
 					timestamp: this.model.get('mtime'),
 					date: OC.Util.relativeModifiedDate(this.model.get('mtime')),
 					hasFavoriteAction: hasFavoriteAction,
-					starAltText: isFavorite ? t('files', 'Favorited') : t('files', 'Favorite'),
+					starAltText: isFavorite ? t('files', 'Favoured') : t('files', 'Favorite'),
```
Comment (nickvergessen) [inline apps/files/js/mainfileinfodetailview.js:166]:
```suggestion
					starAltText: isFavorite ? t('files', 'Favoured') : t('files', 'Favour'),
```
But might need a translators hint then to make clear it's a "verb"?
https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html#provide-context-hints-for-translators

File: apps/files/js/mainfileinfodetailview.js [hunk 2]
```diff
@@ -159,7 +159,8 @@
 					timestamp: this.model.get('mtime'),
 					date: OC.Util.relativeModifiedDate(this.model.get('mtime')),
 					hasFavoriteAction: hasFavoriteAction,
-					starAltText: isFavorite ? t('files', 'Favorited') : t('files', 'Favorite'),
+					// TRANSLATORS "Favour" is a verb
+					starAltText: isFavorite ? t('files', 'Favoured') : t('files', 'Favour'),
```
Comment (nickvergessen) [inline apps/files/js/mainfileinfodetailview.js:163]: IIRC this does not work well. the translator hint is only used on the first translation, but maybe it was only in vue files

**Review by nickvergessen — APPROVED** [review]

**Review by Altahrim — APPROVED** [review]

**Review by miaulalala — CHANGES_REQUESTED** [review]

**Comment by nickvergessen** [comment]
> Conflicting files
> - apps/files/js/mainfileinfodetailview.js
> - apps/files/js/tagsplugin.js 

Both got deleted. So if we want to backport we should copy this commit to stable30 and redo the PR for master

**Pull request merged**